extra/src/main/java/aima/extra/search/pqueue/uninformed/TreePriorityQueueSearch.java

```diff
@@ -42,16 +43,16 @@
 		// node <- a node with STATE = problem.INITIAL-STATE
 		Node<A, S> node = newRootNode(problem.initialState());
 		// frontier <- a priority queue, with node as the only element
-		Queue<Node<A, S>> frontier = newFrontier(node);
+		PriorityQueue<Node<A, S>> pqFrontier = newFrontierPriority(node, getNodeComparator());

```

Comment:
This approach missed the need for the same change to be made to the GraphRLPriorityQueueSearch implementation as well.